examples/fluids/navierstokes.c

```diff
@@ -45,11 +45,14 @@
 const char help[] = "Solve Navier-Stokes using PETSc and libCEED\n";
 
 #include "navierstokes.h"
+#include <ceed/sycl.h>
```

Comment:
Probably should guard this include in case it requires sycl specific headers in the future

Reply:
Wouldn't that contradict with putting `CeedGetStream_Sycl ... __attribute__((weak))` into that header then?

Reply:
`CeedGetStream_Sycl ...__attribute__((weak))` is there to prevent errors at link time when the Sycl backend isn't compiled (and thus the actual implementation for `CeedGetStream_Sycl` doesn't exist). If we put `CeedGetStream_Sycl ...__attribute__((weak))` into `ceed/sycl.h` and include guards, then instances without a sycl backend will still have linking errors (because `CeedGetStream_Sycl` wouldn't be defined anywhere anymore).

Reply:
Ah, I see. Skip the guard and we'll handle the future when it gets here?

Reply:
Sounds fine by me. Not sure if this can/should be generalized to include other backends (if the concept exists of course).